tasks/main.yml

```
@@ -1,2 +1,20 @@
 ---
-# tasks file for skeleton
+- name: Download Metasploit Framework Installer
+  become: true
+  ansible.builtin.get_url:
+    url: https://raw.githubusercontent.com/rapid7/metasploit-omnibus/master/config/templates/metasploit-framework-wrappers/msfupdate.erb
+    dest: /tmp/msfinstall
+    mode: 0755
+  register: metasploit
+
+- name: Install Metasploit Framework
+  when: metasploit.changed
+  become: true
+  ansible.builtin.command: /tmp/msfinstall
```

- `become: true` is the default, so you need not specify it.
- There is no harm in re-downloading the installer, and doing so avoids triggering the "handler" warning from `ansible-lint`.
```suggestion
- name: Download Metasploit Framework Installer
  ansible.builtin.get_url:
    url: https://raw.githubusercontent.com/rapid7/metasploit-omnibus/master/config/templates/metasploit-framework-wrappers/msfupdate.erb
    dest: /tmp/msfinstall
    mode: 0755

- name: Install Metasploit Framework
  ansible.builtin.command: /tmp/msfinstall
```